content/_data/books/Mastering_Jenkins.yml

```
@@ -0,0 +1,19 @@
+title: "Mastering Jenkins"
+url: https://www.amazon.in/Mastering-Jenkins-Jonathan-McAllister/dp/1784390895/ref=sr_1_6?crid=3CUHJHQY7X09V&keywords=jenkins+books&qid=1676393236&sprefix=jenkins+books%2Caps%2C241&sr=8-6
+author: "Jonathan McAllister"
+publication_year: 2015
+publisher: "Ingram short title"
```

As far as I can tell, this book is published by Packt Publishing rather than Ingram
```suggestion
publisher: "Packt Publishing"
```